transport_nantes/mailing_list/forms.py

```diff
@@ -68,19 +68,35 @@ class SubscribeUpdateForm(Form):
 # First step of quick signup to the newsletter
 
 
+def validate_mailinglist_name(mailinglist_token):
+    """Validate the mailing list name.
+
+    """
+    if not MailingList.objects.filter(
+            mailing_list_token=mailinglist_token).exists():
+        raise forms.ValidationError(
```

Comment:
Does the validation handler catch this for you and turn it into "invalid"?

Reply:
If we raise form.ValidationError, the form is considered Invalid and form_invalid() is called, if this is your question ?

Reply:
Yup, just didn't remember precisely and wanted to be sure it was caught!